include/libopencm3/stm32/f0/dac.h

```diff
@@ -33,37 +33,20 @@
 #ifndef LIBOPENCM3_DAC_H
 #define LIBOPENCM3_DAC_H
 
-#include <libopencm3/stm32/common/dac_common_all.h>
+#include <libopencm3/stm32/common/dac_common_v1.h>
 
 /*****************************************************************************/
 /* Module definitions                                                        */
 /*****************************************************************************/
 
 #define DAC				DAC_BASE
```

Comment:
seeing as we're breaking things, do we wnat to move this to DAC1 too?

Reply:
(Or at least, DAC _and_ DAC1?  (we'v edone that with ADC for  many parts already too)

Reply:
I guess so yeah, DAC1 makes more sense as it makes it more obvious that the API is designed to support multiple rather than it being repetitive boilerplate, you agree?

Reply:
Ok, switched to just DAC1, no need for backward compat given we're breaking the API anyway and the DAC parameter was previously unused.